position/position.go

```diff
@@ -0,0 +1,15 @@
+package position
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Position struct {
+	UserID    uuid.UUID // id of user that produced position notification event
+	Latitude  float32   // obviously latitude component of coordinates
+	Longitude float32   // obviously longitude component of coordinates
+	Timestamp time.Time // time when position collected
+	Arrival   time.Time // time when position accepted by system
```

Comment:
Comments should not appear after fields, and should be in format: "FieldName ...".